pkg/framework/aries/framework.go

```diff
@@ -0,0 +1,52 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package aries
+
+import (
+	"github.com/hyperledger/aries-framework-go/pkg/framework/aries/api"
```

Comment:
Is this due to a circular dependency?

Reply:
By "this", I mean the sub-package "api" @rolsonquadras

Reply:
sorry @troyronda missed this comment.

yes, @sudeshrshetty mentioned the possibility of circular dependency (same is done in fabric-sdk-go as well).